.github/workflows/ci-testing.yml

```diff
@@ -19,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        python-version: [ 3.9 ]
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
```

Comment:
3.6 is ut of life cycle and I think it is fine just to test the oldest and latest python version, so 3.7 and 3.9
```suggestion
        python-version: [ 3.7, 3.9 ]
```

Reply:
@Borda thanks for pointing this out! Do you think we should drop official support for 3.6 then? i.e. Update the README and the tutorials to require 3.7?

<img width="861" alt="Screen Shot 2022-01-11 at 10 37 15 AM" src="https://user-images.githubusercontent.com/26833433/149017700-e9ed7c68-cba1-408c-a57e-1f702200df43.png">

Reply:
as py3.6 is out of its life-cycle it is completely reasonable to drop it...

Reply:
@Borda ok got it! I'll try to get that done. I think most of these changes are not in the code but in the tutorials etc. I'll have to coordinate the changes everywhere.